projects/kserve/testing/prepare_kserve.py

```diff
@@ -158,9 +165,11 @@ def undeploy_operator(operator, mute=True):
 def cleanup(mute=True):
     prepare_rhoai.uninstall(mute)
 
-    if not config.project.get_config("kserve.raw_deployment.enabled"):
-        with run.Parallel("cleanup_kserve") as parallel:
-            for operator in config.project.get_config("prepare.operators"):
+    with run.Parallel("cleanup_kserve") as parallel:
+        for operator in config.project.get_config("prepare.operators"):
+           if not config.project.get_config("kserve.raw_deployment.enabled"):
+                undeploy_operator(operator)
+           elif config.project.get_config("kserve.raw_deployment.enabled") and 'raw' in operator['kserve_mode']:
```

Comment:
same comment as above, `undeploy_operator(operator)` is written twice, and `config.project.get_config("kserve.raw_deployment.enabled")`. 
need to find a better way to write this `if/elif` block